libexec/basher-_clone

```diff
@@ -5,18 +5,26 @@
 
 set -e
 
-if [ "$#" -ne 1 ]; then
+if [ "$#" -lt 1 ]; then
   basher-help _clone
   exit 1
 fi
 
-package="$1"
+package=$1
```

Comment:
Wildcard expansion is not performed either for assignments.

e.g.:

    > var_one=*
    > declare -p var_one
    declare -- var_one="*"

"declare -p" shows you the statement which generates the current contents of the variable.  As you can see, "*" was not expanded prior to assignment, otherwise the contents of var_one would have been the directory listing of the current directory.

So if $1 contained a pattern, it would not be expanded as you were worried about:

    func_one() {
    > local var_two=$1
    > declare -p var_two
    > }
    > func_one "*"
    declare -- var_two="*"

As you can see, no expansion due to assignment.
> var_two=$1
    > declare -p var_two
    declare -- two="*"